rust/feature-flags/Cargo.toml

```diff
@@ -39,6 +39,8 @@ health = { path = "../common/health" }
 common-metrics = { path = "../common/metrics" }
 tower = { workspace = true }
 derive_builder = "0.20.1"
+petgraph = "0.6.5"
+moka = { version = "0.12.8", features = ["future"] }
```

Comment:
caching lib with support for TTL and feature weighting

Reply:
As a heads up, this is already in the workspace, you can probably pull it in (we're using it in error tracking).